docs/samplesheets/examples.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Channel usage examples
+description: Examples of advanced sample sheet creation techniques.
+---
+
+# Sample sheet channel manipulation examples
+
+## Separate entries based on a condition
+
+You can use the [`.branch()` operator](https://www.nextflow.io/docs/latest/operator.html#branch) to separate the channel entries based on a condition. This is especially useful when you can get multiple types of input data.
+
+This example shows a channel which can have entries for WES or WGS data. These analysis are different so we want to separate the WES and WGS entries from eachother. We also don't want the `bed` file input for the WGS data, so the resulting channel with WGS data should not have this file in it.
+
+```groovy
+// example channel:
+// [[id:example, type:WGS], WGS.bam, WGS.bam.bai, []]
+// [[id:example2, type:WES], WES.bam, WES.bam.bai, WES.bed]
```

Comment:
```suggestion
// example sample sheet:
// sample, type, bam, bai, bed
// example, WGS, WGS.bam, WGS.bam.bai,
// example2, WES, WES.bam, WES.bam.bai, WES.bed
```

Reply:
It's not really clear in this case what the channel is that comes out of the `fromSamplesheet()` function. This is my opinion on this, but I don't really like this

Reply:
No what you see below is the channel after the `.branch()` manipulation. That part of the docs is showing some examples for how to handle the Channel created by the `.fromSamplesheet()` :)